### DISCUSSION TITLE
Changing the 'Calibre' font will reduce some alignment issues in the UI

### DISCUSSION THREAD

**Discussion by RakeshUP** [snapshot-labs/snapshot#784] [Ideas]
The font 'Calibre' used in snapshot UI has abnormal ascender:descender ratio height as posted in the image below

![Screenshot 2021-09-29 at 12 30 31 PM](https://user-images.githubusercontent.com/20946466/135229123-33a3ffbc-b9cb-4136-9474-6f55bc9011e9.png)

For reference, this is what any font would look like(Roboto from Google fonts used for sample here)

![Screenshot 2021-09-29 at 12 30 13 PM](https://user-images.githubusercontent.com/20946466/135229274-efedb1e7-1fee-430c-911e-e1d3939a0d45.png)

You can check out more about this here https://seek-oss.github.io/capsize/

This abnormal descender height leads to alignment issues, where devs have to align by using a bottom padding/margin of around 4px lesser than the top one to make it look centered. It's also difficult to use center-align in a flexbox. Below is an example of Calibre being center-aligned in a flex

![Screenshot 2021-09-29 at 12 25 47 PM](https://user-images.githubusercontent.com/20946466/135230316-b077a092-b610-41be-b231-b184b458cdfa.png)

The items can be center aligned by specifying a margin(positive/negative) to any one of the items.

A place where alignment is not proper:

![Screenshot 2021-09-29 at 12 22 02 PM](https://user-images.githubusercontent.com/20946466/135230091-22d36ea7-8cdf-4e0a-ad61-0e8d7280548c.png)

This is how it looks with the default system font:

![Screenshot 2021-09-29 at 12 26 43 PM](https://user-images.githubusercontent.com/20946466/135230160-3354c5fc-1b7c-40ab-8f5d-a8f141cdb0e8.png)

The solution would be to change the font (or) get a set of Calibre fonts with proper dimensions. The ones I saw on the official Calibre font website do have the proper dimensions.

**Comment by bonustrack** [comment]
@RakeshUP Yes thats good point, this is something we've seen. We actually got a set of the same font with proper alignement, I can provide it to you but we will need to rework all the UI to be sure nothing break.

**Reply by RakeshUP** [reply]
@bonustrack I can work on that. I'll go through all the pages.

**Reply by samwhosung** [reply]
@RakeshUP great, can't wait to see this!

**Comment by samwhosung** [comment]
@RakeshUP I wan't aware of why this was happening, thanks for the clarification!
